Cargo.toml

```diff
@@ -38,7 +37,7 @@ serde = { version = "1.0", features=["derive"], optional = true }
 
 # This should be an optional dev-dependency (only needed for integration tests),
 # but dev-dependency cannot be optional, and without optionality older toolchain try to compile it and fails
-elementsd = { version="0.3.0", features=["0_21_0","bitcoind_22_0"], optional = true }
+elementsd = {git = "https://github.com/sanket1729/elementsd", rev = "ac9457d1131d32b924cb945a872403c24aa43076", features=["0_21_0","bitcoind_22_0"], optional = true }
```

Comment:
elementsd `0.5.0`  has been released with updated bitcoins with bitcoin `0.28.0`

Reply:
Thanks. Will update the PR